README.md

```diff
@@ -154,13 +154,13 @@ generated code, hence they shouldn't be modified by hand.
 
 #### Automatic Deployment
 
-Run `deploy.py [version]`. See `deploy.py -h` for additional info.
+Run `deploy.py [major|minor|patch]`. See `deploy.py -h` for additional info.
 
 #### Manual Deployment
 
   1. Merge in the desired changes into the `master` branch and commit them.
   2. Clone the repo, work on master.
-  3. Edit package version in `asana/__init__.py` to indicate the [semantic version](http://semver.org/) change.
+  3. Edit package version in `asana/__init__.py` and `VERSION` to indicate the [semantic version](http://semver.org/) change.
```

Comment:
Suggestion: Super minor, but changing `VERSION` to `./VERSION` (or even just linking to [VERSION](https://github.com/Asana/python-asana/blob/master/VERSION)) can help clarify that it's a file in the root directory.

Reply:
Sure thing we can change it to show `./VERSION`. I don't think linking is a good idea since if a user forks the repo the link will take it to this repo's `VERSION` file.